pkg/reconciler/v1alpha1/configuration/configuration.go

```diff
@@ -230,3 +253,59 @@ func (c *Reconciler) updateStatus(u *v1alpha1.Configuration) (*v1alpha1.Configur
 	}
 	return newu, nil
 }
+
+func (c *Reconciler) gcRevisions(ctx context.Context, config *v1alpha1.Configuration) error {
+	logger := logging.FromContext(ctx)
+
+	selector := labels.Set{serving.ConfigurationLabelKey: config.Name}.AsSelector()
+	revs, err := c.revisionLister.Revisions(config.Namespace).List(selector)
+	if err != nil {
+		return err
+	}
+
+	for _, rev := range revs {
```

Comment:
We need to ignore `LastestReadyRevisionName`, which could otherwise satisfy this criteria.

Reply:
this?

Reply:
Added check for this in isRevisionStale